scripts/js_texts/extracted_strings_app.js

```diff
@@ -122,16 +140,60 @@ module.exports = [
     'High',
     'High Barrier',
     'High Tick',
+    'High Tick/Low Tick contracts have a strict duration of five ticks.',
     'High barrier',
+    'High barrier must be higher than low barrier',
+    'High barrier offset',
+    'High, Low and Close',
     'High-Close',
     'High-Low',
     'Higher',
+    'Higher/Lower contracts will be refunded at the purchase price if there are less than 2 ticks between the start and end times.',
     'Hour',
     'I want to reapply',
     'IDK',
+    'If the <strong>exit spot</strong> is equal to either the <strong>Low barrier</strong> or the <strong>High barrier</strong>, you don\'t win the payout.',
```

Comment:
the html tags shouldn't be present here. It should be following the below format[_1], [_2].
![image](https://github.com/user-attachments/assets/5a3e8a62-7283-4d04-a8cf-b0edddb92510)

Reply:
Later putri will be asking for usage of strong 😂

Reply:
@Nuzhy-Deriv  please help updating the strings

Reply:
@balakrishna-deriv @prince-deriv 
there were already existing strings with `<strong>` and traslated. Isn't updating this might cause to wait for new translations?
![image](https://github.com/user-attachments/assets/d82032c8-e20e-41ab-afe4-0a7085330b15)